File: FrameworkManager/sample/packages/AndroidProject/Project/app/src/main/java/jp/co/project/model/UserModel.java
package jp.co.project.model;

import android.content.Context;

import com.unicorn.constant.Constant;

import jp.co.project.model.base.UserModelBase;
/**
 * UserModelはUserテーブルのデータを参照、保存するためのクラスです
 * 
 * @author　c1363
 */
public class UserModel extends UserModelBase {

	public UserModel(Context argContext) {
		super(argContext, Constant.PROTOCOL, Constant.DOMAIN_NAME, Constant.URL_BASE,
				Constant.COOKIE_TOKEN_NAME, Constant.SESSION_CRYPT_KEY, Constant.SESSION_CRYPT_IV);
	}
}

File: FrameworkManager/sample/packages/iOSProject/Project/Class/Model/UserModel.m
//
//  UserModel.m
//  自由に拡張可能です
//
//  Copyright (c) 2014年 saimushi. All rights reserved.
//

#import "common.h"
#import "UserModel.h"

@implementation UserModel

@synthesize profile;

/* オーバーライド */
- (id)init;
{
    self = [super init:PROTOCOL :DOMAIN_NAME :URL_BASE :COOKIE_TOKEN_NAME :SESSION_CRYPT_KEY :SESSION_CRYPT_IV :DEVICE_TOKEN_KEY_NAME :TIMEOUT];
    return self;
}

@end
